cmd/daemon/kubernetes/pods.go

```diff
@@ -274,3 +275,13 @@ func getCodeOwnerSquad(labels map[string]string) string {
 	}
 	return "no-one"
 }
+
+func shouldSquadBeAlerted(squad string, annotations map[string]string) (alertChannel string) {
+	if value, ok := annotations[runtimeAlertsAnnotationKey]; ok {
+		if value == "false" {
+			return "false"
```

Comment:
I think an empty string is better here to avoid an odd string looking like a bool value 🙈
```suggestion
			return ""
```